packages/contracts/contracts/test/L1Block.t.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity 0.8.10;
+
+import { DSTest } from '../../lib/ds-test/src/test.sol';
+import { L1Block } from '../L2/L1Block.sol';
+
+interface CheatCodes {
+  function prank(address) external;
+}
+
+contract L1BLockTest is DSTest {
+
+    CheatCodes cheats = CheatCodes(HEVM_ADDRESS);
+    L1Block lb;
+    address depositor;
+    bytes32 immutable NON_ZERO_HASH = keccak256(abi.encode(1));
+
+    function setUp() external {
+        lb = new L1Block();
+        depositor = lb.DEPOSITOR_ACCOUNT();
+        cheats.prank(depositor);
```

Comment:
`prank` sets the passed address as the sender of any subsequent call?

Reply:
Correct